### ISSUE TITLE
Webman not showing any folder, but manage to open vsh menu

### ISSUE THREAD

**Issue by dannwonn** [aldostools/webMAN-MOD#961]
Hello!! Today i installed overflow cfw cobra to my ps3, but i see no webman folder showing up, when i restart, there is a notification saying webman version thingy, saying that it start up.

but when i hold select, i manage to open the vsh menu for webman, i selected 8.webman setup and then turn on webman games and checked disable content check when startup, but still no result. i also tried reinstalling the firmware, and the webman mod

please helpp!

PS3 Slim Overflow v5 4.84 CFW

**Comment by aldostools** [comment]
Please set the memory for for the plugin to the maximum and verify that the option for Disable content scanning at startup is checked in /setup.ps3. If it isn't check it and save the settings.

The content scanning may take a few minutes depending the number of files to be scanned. Sometimes a file with non ANSI characters may break the XML.

Please try disconnecting all USB devices and scan only the files in the internal HDD. To scan the games use SELECT+L3 or run prepISO or select Refresh XML + XMB from webMAN Setup menu on XMB.

If webMAN MOD Games menu appears, you will have to verify the file names in your USB devices. If the the menu still doesn't appear, some game stored in the internal HDD is causing the issue.

**Comment by dannwonn** [comment]
Thanks for replying,
i tried all methods above, setting the plugin memory to max, unplugging all usb, checking disable content scanning at startup, reloading xmb (3 times), and still no result, i'll attach some photos of my configuration.

![20231008_130708](https://github.com/aldostools/webMAN-MOD/assets/147295373/a3448060-38ac-46e6-bbf2-d4624f010c87)

![20231008_130730](https://github.com/aldostools/webMAN-MOD/assets/147295373/2640cf97-b2be-404f-9919-1aa761536e19)
![20231008_130749](https://github.com/aldostools/webMAN-MOD/assets/147295373/df1d18f6-b21e-4b56-b2fd-65d66468019c)
![20231008_130646](https://github.com/aldostools/webMAN-MOD/assets/147295373/6b8a19d4-9fae-4de4-973f-2455fe509ac6)

**Comment by dannwonn** [comment]
Sorry, accidentally press close issue.

**Comment by aldostools** [comment]
Try unchecking the option Disable grouping of content in "webMAN Games".

Then save settings and rescan again. That option may not be working properly.

**Comment by dannwonn** [comment]
Still no results.

**Comment by aldostools** [comment]
> Still no results.

You could try deleting /dev_hdd0/tmp/wm_config.bin to reset the settings to the default values. You will need to restart the console.

Another option to explore is using slaunch game menu pressing L2+R2 or pressing START for a few seconds.

Honestly I'm running out of ideas about what could be wrong with your system

**Comment by dannwonn** [comment]
> > Still no results.
> 
> You could try deleting /dev_hdd0/tmp/wm_config.bin to reset the settings to the default values. You will need to restart the console.
> 
> Another option to explore is using slaunch game menu pressing L2+R2 or pressing START for a few seconds.
> 
> Honestly I'm running out of ideas about what could be wrong with your system

yep, still didn't work, maybe because an incompatible cfw firmware? (overflow)

**Comment by aldostools** [comment]
Maybe. I only have tested on Rebug and Evilnat custom firmwares.

**Comment by dannwonn** [comment]
> Maybe. I only have tested on Rebug and Evilnat custom firmwares.

hmm, maybe i'll try installing either of them later and let you know, thanks for your help!
